### PR TITLE
remove GA, replace with GTM; resolves #1566

### LINKED ISSUE #1566: Add Google Tag Manager

### PR DESCRIPTION
**Pull request by kschzt**

### CODE REVIEW

File: views/editor.handlebars [hunk 1]
```diff
@@ -285,7 +285,12 @@
 				email: '{{user.email}}'
 			})
 
-			mixpanel.identify('{{user._id}}')
+			dataLayer.push({
+				event: 'User info',
+				username: '{{user.username}}',
+				name: '{{user.name}}',
+				email: '{{user.email}}'
```
Comment (gmarinov) [inline views/editor.handlebars]: do we need to send the email and name? would be nice not to play with this fire.

**Comment by kkananen** [comment]
while transitioning over to GTM, would it be good to track both mixpanel and gtm for some time, to verify the data against each other?

**Comment by kkananen** [comment]
lgtm

**Comment by kkananen** [comment]
lgtm

**Pull request merged**